apps/meteor/app/utils/server/functions/getMongoInfo.ts

```diff
@@ -1,13 +1,9 @@
 import { MongoInternals } from 'meteor/mongo';
 
-import { isWatcherRunning } from '../../../../server/modules/watchers/watchers.module';
-
 function getOplogInfo(): { oplogEnabled: boolean; mongo: MongoConnection } {
 	const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
 
-	const oplogEnabled = isWatcherRunning();
-
-	return { oplogEnabled, mongo };
+	return { oplogEnabled: true, mongo };
```

Comment:
maybe add a TODO here or explain it is this way

Reply:
better I removed the whole usage